.github/workflows/continuous.yml

```diff
@@ -145,6 +145,7 @@ jobs:
       - name: Compile PaddlePaddle JNI
         run: ./gradlew :paddlepaddle:paddlepaddle-native:compileJNI
       - name: Integration tests
+        # MXNet integration test on Github Action fail due to memory limitation on Windows
         run: |
-          ./gradlew :paddlepaddle:paddlepaddle-engine:test :paddlepaddle:paddlepaddle-model-zoo:test
```

Comment:
This will disable all paddle and onnx test coverage.

Reply:
1. We are not collecting test coverage here, this run doesn't runt jRR
2. gradle build will run all tests except MXNet integration test